Assets/Modules/Hero/Scripts/HeroInstantier.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Aloha.Hero
+{
+    public class HeroInstantier : Singleton<HeroInstantier>
+    {
+        [SerializeField] private List<GameObject> HeroPrefabs;
+
+        public GameObject InstantiateHero(int id)
+        {
+            GameObject instance = Instantiate(HeroPrefabs[id]);
+            return instance;
+        }
+
+        public GameObject InstantiateHero(HeroType type)
+        {
+            return InstantiateHero((int)type);
+        }
+    }
+    public enum HeroType
+    {
+        generic = 0,
+        guerrier = 1
```

Comment:
```suggestion
        Generic = 0,
        Guerrier = 1
```
Norme avec majuscule (cf : [C# Doc](https://docs.microsoft.com/fr-fr/dotnet/csharp/language-reference/builtin-types/enum))